From 708cedf4500508ca1a1a1f881a7543896d36829b Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Sun, 20 Dec 2020 19:31:49 +0200
Subject: [PATCH] [SW-0] scripts: ssh and scp with no passwords

We stoped using passwords in code, ssh connections and scp will be done
by ssh keys only. The script should not copy files from nuc to host,
this done by Robot before calling the script

Change-Id: I351c1c0283c80fea56fc55e6ef6cdd4638eef8b7
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
---

diff --git a/scripts/tests/remote_run b/scripts/tests/remote_run
index a92c85a..0ea9979 100755
--- a/scripts/tests/remote_run
+++ b/scripts/tests/remote_run
@@ -11,12 +11,9 @@
 bmc_disabled=$(echo $params | grep -ow bmc_disabled | uniq)
 
 result=~/$(basename $0).result
-fw_dir=/lib/firmware/habanalabs/$device
 local_scr_dir=/var/lib/tftpboot/tests
 scr_name=(local_run local_remote_common)
-fw_ldr=hl-fw-loader
 remote_scr_dir="."
-HL_PASSWD=${HL_ROOT_PASS:-123456}
 
 ssh_flags="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR"
 
@@ -30,30 +27,6 @@
 fi
 
 remote_args="$cmd_prepare $cmd_build $cmd_program $bmc_disabled"
-
-function remote_prepare_work_dirs() {
-	ssh $ssh_flags -t $host "sudo mkdir -p $fw_dir;" | tee -a $result
-
-	local pipestat=${PIPESTATUS[0]}
-	print_msg "ssh mkdir $fw_dir status: $pipestat" | tee -a $result
-
-	return $pipestat;
-}
-
-function remote_copy_fw_scripts() {
-	local scripts=(${scr_name[*]/local_/${local_scr_dir}/local_})
-	# sync the latest built binaries
-	for ((j=0; j<3; j++)); do
-		sshpass -p${HL_PASSWD} scp $ssh_flags -p ${fw_dir}/* root@${host}:${fw_dir}/			|| break;
-		sshpass -p${HL_PASSWD} scp $ssh_flags -p ${local_scr_dir}/${fw_ldr} ${host}:${remote_scr_dir}/	|| break;
-		sshpass -p${HL_PASSWD} scp $ssh_flags -p ${scripts[*]} ${host}:${remote_scr_dir}/		|| break;
-		print_msg "Finished copying the firmware and scripts" | tee -a $result
-		return 0;
-	done
-
-	print_msg "Failed copying the firmware and scripts" | tee -a $result
-	return 1;
-}
 
 function remote_run_script() {
 	print_msg "Starting ${scr_name[0]}, args: $remote_args" | tee -a $result
@@ -90,8 +63,6 @@
 	echo $i
 	sleep 1;
 	ping -c 1 -W 1 $host		|| continue;
-	remote_prepare_work_dirs	|| continue;
-	remote_copy_fw_scripts		|| break;
 	remote_run_script		|| break;
 	echo done;
 	exit 0;
